### DISCUSSION TITLE
Not in the Models Registry

### DISCUSSION THREAD

**Discussion by smcs88** [obss/sahi#651] [Q&A] [answered]
Hello, 

I am trying to perform sliced prediction using "CBRes2Net" from https://github.com/shinya7y/UniverseNet which is based on the format of mmdetection. Is there any option to perform prediction using models which are currently not supported in the original mmdetection?

**Comment by fcakyon** [comment, marked as answer]
Hello @smcs88, `sahi` can support any detector following these steps: https://github.com/obss/sahi#contributing

Check this repo on how to implement zero shot object detector with `sahi`: https://github.com/machinelearnear/large-scale-object-detection-with-sahi-detectron2

**Comment by smcs88** [comment]
Thank you @fcakyon for the quick response and the support!
